<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WizUtility</class>
 <widget class="QMainWindow" name="WizUtility">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizUtility</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="attachButton">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Attach to Game</string>
    </property>
   </widget>
   <widget class="QPushButton" name="questPortButton">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>224</y>
      <width>141</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Teleport to Current Quest</string>
    </property>
   </widget>
   <widget class="QLabel" name="coordsLabel">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>39</y>
      <width>64</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Coordinates</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="xCoordEdit">
    <property name="geometry">
     <rect>
      <x>22</x>
      <y>61</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="yCoordEdit">
    <property name="geometry">
     <rect>
      <x>22</x>
      <y>115</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="zCoordEdit">
    <property name="geometry">
     <rect>
      <x>22</x>
      <y>170</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>61</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>115</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Y</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>170</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Z</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
   </widget>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
